test/integration/create-community-controller.spec.ts

```diff
@@ -268,6 +269,32 @@ test('Create Community Controller', async function ({ components, stubComponents
               expect(await response.json()).toMatchObject({ message: 'Failed to fetch names' })
             })
           })
+
+          describe('and the community privacy is private', () => {
+            let validBodyWithPrivatePrivacy = {
+              ...validBody,
+              privacy: CommunityPrivacyEnum.Private
+            }
+
+            it('should create community with private privacy', async () => {
```

Comment:
Just to be a bit closer to the testing standards, to make them clearer, `it`s should refer to the expectations:
```suggestion
            it('should respond with a 201 and the created community with private privacy', async () => {
```